pupil-spa/src/app/familiarisation-area/familiarisation-area.component.spec.ts

```diff
@@ -56,15 +56,14 @@ describe('FamiliarisationAreaComponent', () => {
       component = fixture.componentInstance;
       fixture.detectChanges();
     }));
-    it('should create', () => {
+    it('should expected the component to exist', () => {
```

Comment:
small typo `expect` instead of `expected`

Reply:
@GuyHarwood thanks, it's updated